### ISSUE TITLE
Strange behavior for single-digit minutes in firefox

### ISSUE THREAD

**Issue by georgesimms** [ocadotechnology/hexagonjs#147]
![screenshot from 2016-02-22 13 06 18](https://cloud.githubusercontent.com/assets/7927477/13219017/4af22b4e-d965-11e5-9eb1-06a5b82b9d67.png)

I have firefox v44.0.2

**Comment by jmsmyth** [comment]
@georgesimms Just to clarify what the strange behaviour is - are you expecting it to show '06' in the minute number picker?

**Comment by georgesimms** [comment]
Yes. Also that's what it does in Chrome, so it should be at very least consistent.

**Comment by c-frater** [comment]
It looks like Chrome is actually 'wrong' in this instance. 
As far as I can see, the spec says that an `input` of type `number` should be a `floating point number` which would indicate that it _shouldn't_ have a leading zero.
The suggested solution I've seen is to use `<input type="text" pattern="[0-9]*" />` which will apparently retain the explicit entering of numbers and the 'number' keyboard on mobile devices but I'm not keen on doing this just to make the leading 0 appear for time pickers. Thoughts @jmsmyth @georgesimms ?
